root/static/scripts/edit/utility/linkPhrase.js

```diff
@@ -17,9 +17,13 @@ function mapNameToID(result, info, id) {
     result[info.attribute.name] = id;
 }
 
+const attributeIdsByName = {};
+
 export const stripAttributes = _.memoize(function (linkTypeID, backward) {
     var linkType = linkTypeInfo.byId[linkTypeID];
-    var idsByName = _.transform(linkType.attributes, mapNameToID);
+    var idsByName = attributeIdsByName[linkTypeID] ||
+        (attributeIdsByName[linkTypeID] =
```

Comment:
Not really, I think I was trying to remove the `memoize` for some reason. I've removed that commit since the function is reimplemented later (by calling `interpolate` with an empty set of attributes).